.circleci/config.yml

```diff
@@ -72,3 +82,50 @@ workflows:
           filters:
             branches:
               only: master
```

Comment:
That should be probably dropped.

Name should be changed to reflect that. E.g. `push-cluster-operator-to-aliyun-tag`